mkdocs.yml

```diff
@@ -8,17 +8,19 @@ edit_uri: edit/dev/docs/
 nav:
   - Getting started:
     - Introducing episb: README.md
-  - How-to guides:
+  - User guides:
     - Query types explained: howto-query-provider.md
     - Add a new provider to a hub: howto-subscribe.md
     - Normalize to a segmentation: howto-normalize.md
+  - Administrator guides:
     - Build a data provider: howto-build-provider.md
     - Build a hub: howto-build-hub.md
+  - Programmer's guides:
```

Comment:
I like where you're going with this, but given there's only 1 programmer guide, let's just leave it under User guides... programmers are users, too.

Reply:
Done :smile: